src/plugins/data/server/search/search_service.ts

```diff
@@ -77,7 +81,7 @@ type StrategyMap = Record<string, ISearchStrategy<any, any>>;
 
 /** @internal */
 export interface SearchServiceSetupDependencies {
-  registerFunction: AggsSetupDependencies['registerFunction'];
+  expressions: ExpressionsServerSetup;
```

Comment:
Updated setup dependencies for consistency with the `public` search service, and also as we need `registerType` from the `expressions` contract now.